plugin/src/main/kotlin/com/automattic/android/PublishToS3BaseExtension.kt

```diff
@@ -1,6 +1,6 @@
 package com.automattic.android.publish
 
-interface PublishToS3PluginBaseExtension {
+interface PublishToS3BaseExtension {
```

Comment:
Out of curiosity, what's the rationale behind this particular `interface` rename?

Reply:
It's been a while but iirc I did this to simplify the name as it was getting a bit too verbose switching between files in vim.